docs/Serial.md

```diff
@@ -56,86 +56,156 @@ e.g. after configuring a port for GPS enable the GPS feature.
+```
 
+| Serial cli command arguments |
+| ---------------------------- |
+| 1. Serial Port Identifier    |
+| 2. Serial Port Function      |
+| 3. MSP baud rate             |
+| 4. GPS baud rate             |
+| 5. Telemetry baud rate       |
+| 6. Blackbox baudrate         |
+
+Note: for Identifier see serialPortIdentifier_e in the source; for Function bitmask see serialPortFunction_e in the source code.
+
+### 1. Serial Port Identifier
+
+| Identifier                 | Value |
+| -------------------------- | ----- |
+| SERIAL_PORT_NONE           | -1    |
+| SERIAL_PORT_USART1         | 0     |
+| SERIAL_PORT_USART2         | 1     |
+| SERIAL_PORT_USART3         | 2     |
+| SERIAL_PORT_UART4          | 3     |
+| SERIAL_PORT_UART5          | 4     |
+| SERIAL_PORT_USART6         | 5     |
+| SERIAL_PORT_USART7         | 6     |
+| SERIAL_PORT_USART8         | 7     |
+| SERIAL_PORT_LPUART1        | 8     |
+| SERIAL_PORT_USB_VCP        | 20    |
+| SERIAL_PORT_SOFTSERIAL1    | 30    |
+| SERIAL_PORT_SOFTSERIAL2    | 31    |
+
+### 2. Serial Port Function
+
+| Function                     | Value |
+| ---------------------------- | ----- |
+| FUNCTION_NONE                | 0     |
+| FUNCTION_MSP                 | 1     |
+| FUNCTION_GPS                 | 2     |
+| FUNCTION_TELEMETRY_FRSKY_HUB | 4     |
+| FUNCTION_TELEMETRY_HOTT      | 8     |
+| FUNCTION_TELEMETRY_LTM       | 16    |
+| FUNCTION_TELEMETRY_SMARTPORT | 32    |
+| FUNCTION_RX_SERIAL           | 64    |
+| FUNCTION_BLACKBOX            | 128   |
+| FUNCTION_TELEMETRY_MAVLINK   | 512   |
+| FUNCTION_ESC_SENSOR          | 1024  |
+| FUNCTION_VTX_SMARTAUDIO      | 2048  |
+| FUNCTION_TELEMETRY_IBUS      | 4096  |
+| FUNCTION_VTX_TRAMP           | 8192  |
+| FUNCTION_RCDEVICE            | 16384 |
+| FUNCTION_LIDAR_TF            | 32768 |
+| FUNCTION_FRSKY_OSD           | 65536 |
+
+Note: `FUNCTION_FRSKY_OSD` = `(1<<16)` requires 17 bits.
+
+### 3. MSP Baudrates
+
+| Baudrate |
+| -------- |
+| 9600     |
+| 19200    |
+| 38400    |
+| 57600    |
+| 115200   |
+| 230400   |
+| 250000   |
+| 500000   |
+| 1000000  |
+
+### 4 GPS Baudrates
+
+| Baudrate |
+| -------- |
+| 9600     |
+| 19200    |
+| 38400    |
+| 57600    |
+| 115200   |
+
+Note: Also has a boolean AUTOBAUD. It is recommended to use a fixed baudrate. Configure GPS baudrate according to device documentation.
+
+### 5. Telemetry Baudrates
+
+| Baudrate |
+| -------- |
+| AUTO     |
+| 9600     |
+| 19200    |
+| 38400    |
+| 57600    |
+| 115200   |
+
+### 6. Blackbox Baudrates
+
+| Baudrate |
+| -------- |
+| 19200    |
+| 38400    |
+| 57600    |
+| 115200   |
+| 230400   |
+| 250000   |
+| 400000   |
+| 460800   |
+| 500000   |
+| 921600   |
+| 1000000  |
+| 1500000  |
+| 2000000  |
+| 2470000  |
+
+Note: As Openlog is superseeded by Openlager the 230400 baudrate setting is too slow to be considered useful.
+Other pheripherals like VTX, Camera, LIDAR and OSD (FrSky Protocol) can be configured using this parameter.
```

Comment:
Not sure what this is meant to mean? What can be configured?

Reply:
Yes you are right. I should reduce and keep to the point. Those details belong to other documents like blackbox.MD, osd.MD, VTX.MD. Lines are removed.